graylog2-web-interface/src/stores/alarmcallbacks/AlarmCallbacksStore.js

```diff
@@ -12,24 +12,16 @@ const AlarmCallbacksStore = Reflux.createStore({
   listenables: [AlarmCallbacksActions],
   types: undefined,
 
-  init() {
-    this.available(undefined).then((types) => {
-      this.types = types;
-      this.trigger({types: types});
-    });
-  },
-
   getInitialState() {
     return {
       types: this.types,
```

Comment:
With the changes the store state is not used any more, at least I couldn't find any usages.